#!/usr/bin/env python
PACKAGE = "racecar_learning"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("latency",  double_t,    0, "Latency compensation for control", 0.2, -1, 1)
gen.add("ref_speed",  double_t,    0, "Reference speed for control", 1, 0, 1.5)

exit(gen.generate("racecar_learning", "racecar_learning", "controller"))